### PR TITLE
feat: cover

### PR DESCRIPTION
**Pull request by bxbdev**
## Why need this change? / Root cause:

- Added Cover component
- Added test
- Added storybook

## Changes made:

- Delete previous feature/cover

## Test Scope / Change impact:

-

## Issue

- Closed #49

### CODE REVIEW

File: shared/components/Cover/index.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
```
Comment (JohnsonMao) [inline shared/components/Cover/index.tsx:1]: useEffect 沒有使用到，幫我移除，感謝~

Comment (bxbdev) [reply shared/components/Cover/index.tsx]: 拿掉了，謝謝提醒。

File: shared/components/Cover/Cover.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    width: { control: "number" },
+    height: { control: "number" },
+    fill: { control: "boolean" },
+    className: { control: "text" },
+  },
+};
+
+type Story = StoryObj<typeof Cover>;
+
+export const CoverDefault: Story = {
```
[the remark this answers is not in the document]
Comment (bxbdev) [reply shared/components/Cover/Cover.stories.tsx]: 修改完畢！

File: shared/components/Cover/index.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          fill,
+        }
+      : {
+          width,
+          height,
+        };
+  };
+
+  return (
+    <>
```
[the remark this answers is not in the document]
Comment (JohnsonMao) [reply shared/components/Cover/index.tsx]: 影響不大，但主要是程式碼可讀性

**Review by HongLinLiao — APPROVED** [review]

**Review by Parkerhiphop — APPROVED** [review]

**Pull request merged**